monkey/monkey_island/cc/services/attack/test_mitre_api_interface.py

```diff
@@ -0,0 +1,15 @@
+from unittest import TestCase
+
+from monkey_island.cc.services.attack.mitre_api_interface import MitreApiInterface
+
+
+class TestMitreApiInterface(TestCase):
+
+    def test_get_all_mitigations(self):
+        mitigations = MitreApiInterface.get_all_mitigations()
+        self.assertTrue((len(mitigations.items()) >= 282))
+        mitigation = next(iter(mitigations.values()))
+        self.assertEqual(mitigation['type'], "course-of-action")
+        self.assertTrue(mitigation['name'])
+        self.assertTrue(mitigation['description'])
+        self.assertTrue(mitigation['external_references'])
```

Comment:
![image](https://user-images.githubusercontent.com/48879847/78114850-9108f600-740a-11ea-8a76-761cefc64e81.png)

Reply:
it took me too many tried to get google to say that and totally not worth the joke. but seriously perhaps do this:
```suggestion
        self.assertIsNotNone(mitigation['name'])
        self.assertIsNotNone(mitigation['description'])
        self.assertIsNotNone(mitigation['external_references'])
```

Reply:
Lol. Next time change the source :D regarding comment I'm not sure what would happen if any of these fields are empty and not null. Nothing good probably. But I'll change it nonetheless